index.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+var bodyParser = require('body-parser');
+var express = require('express');
+var GitHubServer = require('./lib/GitHubServer');
+var gitHubWebHook = require('express-github-webhook');
+var nconf = require('nconf');
+var Promise = require('bluebird');
+var RegexTools = require('./lib/RegexTools');
+
+var app = express();
+module.exports = app;
+
+nconf.env('__')
```

Comment:
Ok. I think a good implementation will keep `nconf` in one file, but if that changes I'll abstract it to another file